SimpleRestaurantApi.md

```diff
@@ -87,10 +87,11 @@ to add, remove, and query menu items for each table.  Please make this as simple
 * The client (the restaurant staff “devices” making the requests) MUST be able to: add one or more items with a 
 table number, remove an item for a table, and query the items still remaining for a table.
 * The application MUST, upon creation request, store the item, the table number, and how long the item will take to cook.
-* The application MUST, upon deletion request, remove the item for the table number.
-* The application MUST, upon query request, show all items for the requested table number.
+* The application MUST, upon deletion request, remove a specified item for a specified table number.
+* The application MUST, upon query request, show all items for a specified table number.
```

Comment:
what's the difference of line 91 and 92?

Reply:
so all items vs a specific item? meaning if you ask about the cheese dish table 7 ordered vs all the items table 7 ordered...unless i'm missing something